app/src/test/java/io/swagslash/settlersofcatan/TestPieces.java

```diff
@@ -0,0 +1,87 @@
+package io.swagslash.settlersofcatan;
+
+import junit.framework.Assert;
+
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import io.swagslash.settlersofcatan.pieces.utility.AxialHexLocation;
+import io.swagslash.settlersofcatan.pieces.utility.EdgeDirection;
+import io.swagslash.settlersofcatan.pieces.utility.HexPoint;
+import io.swagslash.settlersofcatan.pieces.utility.HexPointPair;
+import io.swagslash.settlersofcatan.pieces.utility.HexUtility;
+import io.swagslash.settlersofcatan.pieces.utility.VertexDirection;
+
+/**
+ * Created by thoma on 11.06.2018.
+ */
+
+public class TestPieces {
+
+
+    @Test
+    public void TestAxialLocation() {
+        AxialHexLocation axialHexLocation = new AxialHexLocation();
+        axialHexLocation = AxialHexLocation.addAxial(new AxialHexLocation(1, 1), new AxialHexLocation(2, 2));
+        Assert.assertEquals(axialHexLocation.q, 3);
+        Assert.assertEquals(axialHexLocation.r, 3);
+
+        Assert.assertEquals(1, AxialHexLocation.complementAxialDirection(10));
+
+        AxialHexLocation axialHexLocation1 = AxialHexLocation.axialDirection(3);
+
+        axialHexLocation1 = AxialHexLocation.axialNeighbor(axialHexLocation, 2);
+        List<AxialHexLocation> axialHexLocationList = new ArrayList<AxialHexLocation>();
+        axialHexLocationList.add(axialHexLocation1);
+        boolean boo = AxialHexLocation.isCloseTo(axialHexLocation, axialHexLocationList);
+        Assert.assertEquals(true, boo);
+        axialHexLocationList = new ArrayList<AxialHexLocation>();
+        axialHexLocationList.add(new AxialHexLocation(1, 1));
+        boo = AxialHexLocation.isCloseTo(new AxialHexLocation(10, 1), axialHexLocationList);
+        Assert.assertEquals(false, boo);
+
+    }
+
+
+    @Test
+    public void TestCubicLocation() {
+        /*
+        CubicHexLocation cannot be tested, becouse this class is not public
+         */
+    }
+
+    @Test
+    public void TestEdgeDirection() {
+        EdgeDirection e;
+       /*
+       Also private Construktor
```

Comment:
Bin mir nit sicher dass man
a) Dinge nicht testen kann wenn sie an private Constructor haben
b) EdgeDirection unbedingt testen muss, das is a human readable Form von einer Richtung eines Hex Punkts.